fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -98,7 +104,10 @@ std::unordered_map<std::string, KeyRange> SpecialKeySpace::managementApiCommandT
 	      .withPrefix(moduleToBoundary[MODULE::MANAGEMENT].begin) }
 };
 
-std::set<std::string> SpecialKeySpace::options = { "excluded/force", "failed/force" };
+std::set<std::string> SpecialKeySpace::options = { "excluded/force",
```

Comment:
Shall we also change these to `excluded_locality/force`, `failed_locality/force`?